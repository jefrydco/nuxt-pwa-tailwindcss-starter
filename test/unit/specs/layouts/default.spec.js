import { mount, createLocalVue, RouterLinkStub } from '@vue/test-utils'

import Default from '~/layouts/default'
import { div } from '~/test/utils'

const localVue = createLocalVue()

describe('Default layout', () => {
  it('renders to match snapshot', () => {
    const wrapper = mount(Default, {
      localVue,
      stubs: {
        nuxt: div,
        'router-link': RouterLinkStub,
        'nuxt-link': RouterLinkStub,
        'nuxt-child': div
      },
      mocks: {
        $route: {
          query: {}
        }
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
