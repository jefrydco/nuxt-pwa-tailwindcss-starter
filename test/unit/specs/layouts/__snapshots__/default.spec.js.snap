// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout renders to match snapshot 1`] = `
<main class="text-gray-900 leading-normal">
  <div></div>
</main>
`;
